ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -277,6 +278,7 @@ mkNodeArgs registry cfg initState tracers btime chainDB isProducer = NodeArgs
     , blockProduction
     , blockFetchSize      = nodeBlockFetchSize
     , blockMatchesHeader  = nodeBlockMatchesHeader
+    , recentTxIdsExpiryThresh = Time (secondsToDiffTime 3600)
```

Comment:
I'll fix the alignment in another commit :smiley:. Just didn't want to add much `diff` noise to this draft PR.

Reply:
Oh, I hadn't realized